### PR TITLE
Set credentials and proper hostname when updating loopback kubeconfig

### PR DESCRIPTION
**Pull request by sdodson**
Make sure that the user we expect  has proper credentials set.

When a cluster is installed the username in the loopback kubeconfig
is equal to hostname of that local master. This changes the variable
to equal a value of what would originally be set.

Supersedes #10272 #10271 
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1467775#c9

### CODE REVIEW

File: roles/openshift_control_plane/tasks/set_loopback_context.yml [hunk 1]
```diff
@@ -16,6 +16,17 @@
...
 
+- command: >
+    {{ openshift_client_binary }} config set-credentials
+    --client-certificate=/etc/origin/master/openshift-master.crt
+    --client-key=/etc/origin/master/openshift-master.key
+    --embed-certs=true
+    {{ openshift.master.loopback_user }}
+    --config={{ openshift_master_loopback_config }}
+  when:
+  - set_loopback_cluster | changed
```
Comment (michaelgugino) [inline roles/openshift_control_plane/tasks/set_loopback_context.yml]: I think the filter syntax "| changed" is deprecated, should use "is changed" I believe.

**Comment by sdodson** [comment]
wip, working to test this in earlier releases, 3.7 in particular.

**Comment by sdodson** [comment]
This doesn't seem to adversely affect anything in my tests but the customer reporting the problem seems to have a significantly different starting loopback config and I'm no longer confident that this would remedy the problem for them. Also, this isn't included in the upgrade codepath in releases 3.9 and newer. So I'll probably close this all together and focus on fixing release-3.7.

**Comment by sdodson** [comment]
/close
I cannot figure out why travis is failing to fetch the branch, closing and i'll open a new pr.

**Pull request closed without merging**